packages/x-date-pickers/src/CalendarPicker/CalendarPicker.tsx

```diff
@@ -116,6 +117,8 @@ export interface CalendarPickerProps<TDate>
    * @returns {void|Promise} -
    */
   onMonthChange?: (month: TDate) => void | Promise<void>;
+  focusedPicker?: CalendarPickerView | null;
```

Comment:
Shouldn't it be ?

```suggestion
  focusedView?: CalendarPickerView | null;
```

Reply:
Yes, for now it's confusing because we have the `YearPicker`, `monthPicker`, ...  which are in fact views

Reply:
Same for `onFocusedPickerChange`

And yes I agree that `YearPicker` & co are views not pickers